doc/manual.md

```diff
@@ -5013,6 +5013,37 @@ exceptions inherit from `Defect`.
 Exceptions that indicate any other runtime error that can be caught inherit from
 `system.CatchableError` (which is a subtype of `Exception`).
 
+* Exception
+  * CatchableError
+	* IOError
+		* EOFError
+	* OSError
+		* LibraryError
+	* ResourceExhaustedError
+	* ValueError
+		* KeyError
+  * Defect
+	* ArithmeticDefect
+		* DivByZeroDefect
+		* OverflowDefect
+	* AccessViolationDefect
+	* AssertionDefect
+	* OutOfMemDefect
+	* IndexDefect
+	* FieldDefect
+	* RangeDefect
+	* StackOverflowDefect
+	* ReraiseDefect
+	* ObjectAssignmentDefect
+	* ObjectConversionDefect
+	* FloatingPointDefect
+		* FloatInvalidOpDefect
+		* FloatDivByZeroDefect
+		* FloatOverflowDefect
+		* FloatUnderflowDefect
+		* FloatInexactDefect
+	* DeadThreadDefect
+	* NilAccessDefect
```

Comment:
Don't use tabs.

Reply:
I see, I wasn't too familiar with markdown and didn't know using tabs made a code block. I'll use a `tree`-like diagram instead.